setup.py

```diff
@@ -52,33 +52,28 @@
         "google.cloud.documentai_toolbox": ["templates/*.xml.j2"],
     },
     install_requires=(
-        "google-api-core >= 1.31.5, <3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0",
-        "pandas >= 1.0.0, <3.0.0",
-        "pandas >= 1.0.0, <2.0.0; python_version<'3.8'",
-        "pyarrow >= 15.0.0, <16.0.0; python_version>='3.8'",  # Required by Pandas #237
+        "google-api-core >= 2.17.1, <3.0.0dev,!=2.0.*,!=2.1.*,!=2.2.*,!=2.3.0",
+        "pandas[performance,gcp] >= 2.0.0, <3.0.0",
+        "pyarrow >= 15.0.0, <16.0.0",  # Required by Pandas #237
```

Comment:
Please could you update the comment `Required by Pandas #237` with a better description, or remove the comment altogether if `pyarrow` is a direct dependency of this library.

Reply:
pyarrow is a direct dependency of pandas.

Reply:
Could we use the `pyarrow` extra instead? `pandas[gcp,performance,pyarrow] >= 2.0.0, <3.0.0`

https://github.com/pandas-dev/pandas/blob/89b286a699b2d023b7a1ebc468abf230d84ad547/pyproject.toml#L66C1-L66C8